appveyor.yml

```diff
@@ -28,10 +28,12 @@ environment:
 
     - APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2017
       CMAKE_VS_VERSION: "15 2017"
+      CMAKE_OPTIONS: "-DFLATBUFFERS_ENABLE_PCH=1"
       MONSTER_EXTRA: ""
 
     - APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2019
       CMAKE_VS_VERSION: "16 2019"
+      CMAKE_OPTIONS: "-DFLATBUFFERS_ENABLE_PCH=1"
```

Comment:
Maybe have 1 of the 2 build with PCH off, to catch any issues with this system?

Reply:
Isn't it enough with vs 2010 -> 2015 ?